File: tmdb-core/src/main/java/dev/reuss/tmdb/domain/keywords/KeywordPaths.java
package dev.reuss.tmdb.domain.keywords;

import dev.reuss.tmdb.value.id.KeywordId;

final class KeywordPaths {

    private static final String KEYWORD = "/keyword";

    private KeywordPaths() {
    }

    static String details(KeywordId keywordId) {
        return KEYWORD + "/" + keywordId.asString();
    }
}

File: tmdb-core/src/main/kotlin/dev/reuss/tmdb/domain/keywords/KeywordPaths.kt
package dev.reuss.tmdb.domain.keywords

import dev.reuss.tmdb.core.path.tmdbPath
import dev.reuss.tmdb.value.id.KeywordId

internal object KeywordPaths {
    fun details(keywordId: KeywordId): String = tmdbPath("keyword", keywordId)
}
